genesis.go

```diff
@@ -70,6 +70,20 @@ func InitGenesis(ctx context.Context, k *keeper.Keeper, address address.Codec, g
 		panic(errors.Wrap(err, "unable to set genesis stats"))
 	}
 
+	for key, rawAmount := range genesis.TotalExternalYield {
+		provider, identifier := types.ParseYieldRecipientKey(key)
+		amount, ok := math.NewIntFromString(rawAmount)
+		if !ok {
+			panic(fmt.Errorf("unable to parse total external yield %s", rawAmount))
+		}
+
+		key := collections.Join(int32(provider), identifier)
+		err = k.TotalExternalYield.Set(ctx, key, amount)
+		if err != nil {
+			panic(errors.Wrapf(err, "unable to set total external yield (%s/%s:%s)", provider, identifier, amount))
```

Comment:
Let's make this change once CI runs!

```suggestion
			panic(errors.Wrapf(err, "unable to set genesis total external yield (%s/%s:%s)", provider, identifier, amount))
```